statsmodels/tsa/vector_ar/vecm.py

```diff
@@ -609,23 +609,10 @@ def coint_johansen(endog, det_order, k_ar_diff):
                       category=HypothesisTestWarning)
 
     from statsmodels.regression.linear_model import OLS
-    tdiff = np.diff
 
     class Holder(object):
         pass
 
-    def trimr(x, front, end):
```

Comment:
This is only ever called with `end=0`, so the function reduces to the one-liner `x[front:]`.  The reader shouldn't have to look this up.